packages/core/src/components/hotkeys/keyCombo.tsx

```diff
@@ -10,7 +10,8 @@ import { normalizeKeyCombo } from "./hotkeyParser";
 
 const KeyIcons = {
     alt: "pt-icon-key-option",
-    ctrl: "pt-icon-key-control",
+    cmd: "pt-icon-key-command",
+    ctrl: "pt-icon-chevron-up", // "pt-icon-key-ctrl" is unfortunately not vertically centered
```

Comment:
Let's keep using `pt-icon-key-control`, the same one we use for menus à la macOS: ![image](https://user-images.githubusercontent.com/199754/27712769-6128838e-5cdd-11e7-8700-43d4385d39cf.png)

If anything, we should update the icon itself (I can look into it)